sdk/identity/Azure.Identity/src/TenantIdResolver.cs

```diff
@@ -7,20 +7,35 @@ namespace Azure.Identity
 {
     internal static class TenantIdResolver
     {
-        internal const string tenantIdMismatch = "The TenantId received from a challenge did not match the configured TenantId and AllowMultiTenantAuthentication is false.";
+        internal const string tenantIdMismatch =
+            "The TenantId received from a challenge did not match the configured TenantId and AllowMultiTenantAuthentication is false.";
```

Comment:
nit - Looks like this error message is no longer used, if not should we just remove it?